pages/page.py

```diff
@@ -115,15 +97,9 @@ def return_to_previous_page(self):
     def get_relative_path(self, url):
         self.selenium.get(u'%s%s' % (self.base_url, url))
 
-    def find_element(self, *locator):
```

Comment:
They do not align with our style guide. Page objects can simply call the `find_element*` methods directly.